src/components/POLine/POLineView.js

```diff
@@ -110,10 +111,14 @@ class POLineView extends Component {
 
   onEditPOLine = (e) => {
     if (e) e.preventDefault();
-    const { queryMutator } = this.props;
+    const { queryMutator, order, line } = this.props;
 
-    queryMutator.update({ layer: 'edit-po-line' });
-  }
+    queryMutator.update({
+      _path: `/orders/view/${order.id}/po-line/view/${line.id}`,
+      layer: 'edit-po-line',
+      filters: `${FILTERS.STATUS}.${WORKFLOW_STATUS.open},${FILTERS.STATUS}.${WORKFLOW_STATUS.pending}`,
```

Comment:
What the reason you've put filters here?

Reply:
Updated